packages.txt

```diff
@@ -33,7 +33,7 @@ bzip2
 cacert
 ccrypt
 cifs-utils
-# clang
+clang
```

Comment:
This eventually failed to compile as well (it`s clang 4 depending on LLVM 4).

Rest is still compiling but I think they're fine.